sim/hunter/serpent_sting.go

```diff
@@ -93,7 +100,11 @@ func (hunter *Hunter) chimeraShotSerpentStingSpell(rank int) *core.Spell {
 		BonusCoefficient:         0.4,
 
 		ApplyEffects: func(sim *core.Simulation, target *core.Unit, spell *core.Spell) {
-			spell.CalcAndDealDamage(sim, target, baseDamage, spell.OutcomeRangedCritOnly)
+			damage := baseDamage
+			if has6PDragonstalkerPursuit {
+				damage = damage + (0.25 * hunter.GetStat(stats.RangedAttackPower)) / 5
```

Comment:
The way this type of scaling should be applied is with `spell.RangedAttackPower(target)` instead of just getting the stat directly